### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by QY-H00**

### CODE REVIEW

**Review by yhtMinceraft1010X — COMMENTED** [review]
LGTM

**Review by figo2127 — COMMENTED** [review]
Looks good bro!

**Review by AiwassPrime — COMMENTED** [review]
LGTM

File: src/main/java/seedu/module/model/ModuleManager.java [hunk 1]
```diff
@@ -112,6 +112,32 @@ public static void putIntoCorrectWorkloadDistribution(Module module, Task task)
...
+    /**
+     * Modifies the workload distribution after deleting the task.
+     *
+     * @param module the corresponding module of the task
+     * @param task the task that need to be inserted into mapping book.
+     */
+    public static void decreaseCorrectWorkloadDistribution(Module module, Task task) {
+        int workloadLevel = task.getWorkload().getWorkloadLevel();
+        switch(workloadLevel) {
+        case LOW_LEVEL:
```
Comment (umergta) [inline src/main/java/seedu/module/model/ModuleManager.java:124]: nice avoidance of using magic literals!

**Review by umergta — COMMENTED** [review]
LGTM!

**Pull request merged**